apps/src/applab/designElements/ThemeDropdown.jsx

```diff
@@ -0,0 +1,84 @@
+// Dropdown for theme selection that shows sample icons of what
+// each theme will look like
+import PropTypes from 'prop-types';
+import React from 'react';
+import {themeOptionsForSelect} from '../constants';
+import Select from 'react-select';
+import 'react-select/dist/react-select.css';
+
+export default class ThemeDropdown extends React.Component {
+  static propTypes = {
+    initialValue: PropTypes.string.isRequired,
+    handleChange: PropTypes.func.isRequired,
+    desc: PropTypes.node
```

Comment:
NIT [not blocking] while you're here, would you mind updating this prop to `description`? (only worth doing if you have additional changes to make)